src/Nethermind/Nethermind.Blockchain/TxPools/Storages/PersistentTxStorage.cs

```diff
@@ -41,7 +42,7 @@ public Transaction Get(Keccak hash)
 
         public Transaction[] GetAll()
         {
-            var transactionsBytes = _database.GetAll();
+            var transactionsBytes = _database.GetAll().ToArray();
```

Comment:
hmmm

Reply:
As I moved to lazy evaluation in GetAll I didn;t want to break existing code